docs/troubleshooting.adoc

```diff
@@ -0,0 +1,157 @@
+= Troubleshooting Landrush
```

Comment:
@Preeticp Are we going to put other troubleshooting content here too? Looks like the file name is `troubleshooting.adoc` which seems to be general. 

It would be fine right now but we need to put other troubleshooting content then we need to have some sort of categorization i guess.